app/javascript/src/apis/contact.js

```diff
@@ -0,0 +1,39 @@
+const fetch = async () => {
+  const userExample1 = {
+    name: "Ronald Richards",
+    role: "Owner",
+    email: "richards@example.com",
+    profile_pic: "https://i.pravatar.cc/300",
+    created_at: new Date().toString(),
+  };
+  const userExample2 = {
+    name: "Jacob Jones",
+    role: "Owner",
+    email: "jones@example.com",
+    profile_pic: "",
+    created_at: new Date().toString(),
+  };
+  const mock = [];
+  let count = 0;
+  for (let i = 0; i < 50; i++) {
+    userExample1["id"] = count;
+    userExample2["id"] = count + 1;
+    count = count + 2;
+    mock.push(JSON.parse(JSON.stringify(userExample1)));
+    mock.push(JSON.parse(JSON.stringify(userExample2)));
```

Comment:
Not sure why you are doing json stringify and then doing json parse. Isn't it better to directly push the object itself?

Reply:
@navaneethsdk I don't think you need to create a deep copy here. The object is only 1 level nested, right? You could simply spread the object creating a shallow copy.

>Now that I am familiar with Array.from() HOF I have replaced this functionality with the following code snippet:

I believe that's what you have done with the last code block.

Reply:
I see. Yes, it's only 1 level nested so as you said a shallow copy is enough I guess. The last code snippet seems to work fine and produces the desired output Yedhin.